CMSgov/onemac
# replace survey success banner with normal success banner

Story: https://qmacbis.atlassian.net/browse/OY2-21002
Endpoint: See github-actions bot comment

### Details

Replace survey success banner with normal success banner

### Changes

- all success banner messages were configured in one spot in alertlib for both submission and package view

### Test Plan

1. Login as a statesubmitter
2. submit a new package from both submission view and package view
3. verify success banner after submission does not contain the survey language or link
